src/app/shared/services/suggest/suggest.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from '@angular/core';
+import { Jsonp, Response, URLSearchParams } from '@angular/http';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/catch';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/observable/throw';
+
+import { SuggestResponse } from '../../../models/api-response';
+
+@Injectable()
+export class SuggestService {
+	private static apiUrl: URL = new URL('http://api.loklak.org/api/suggest.json');
+	//private static maximum_records_fetch: string = '10';
+	private static minified_results: string = 'true';
```

Comment:
I tried to change this with boolean, but it is throwing this error "Argument of type 'boolean' is not assignable to parameter of type 'string'.", we are receiving it in string form.